week1/MAKEME.md

```diff
@@ -14,30 +14,51 @@ var state = 10;
 
 Endpoints criteria
 ```js
-// /state 
-// response: the current state in a html format 
-// when the server starts, this should return "10"
-http://localhost:8080/state 
-
-// /add
-// Response: "ok" in html format
-// This should add 1 to the current state
-http://localhost:8080/add
-
-// /remove
-// Response: "ok" in html format
-// This should subtract 1 ƒrom the current state
-http://localhost:8080/remove
-
-// /reset
-// Response: "ok" in html format
-// This should set the state back to 10
-http://localhost:8080/reset
-
-// Any other URL
-// Response: return error code 404: Not found with a friendly message
-// and do not change the state variable
-http://localhost:8080/subtract
+'use strict';
+
+// To run this file: node index.js
+// Then in your browser: http://localhost:8080
+let http = require('http');
+let port = 8080;
```

Comment:
if these things are not going to change while the programming is running, why did you use `let` instead of `const`?